content/500-platform/550-platfrom-cli/about.mdx

```diff
@@ -0,0 +1,13 @@
+---
+title: 'About'
+metaTitle: 'Platform CLI: About'
+tocDepth: 3
+---
+
+<TopBlock>
+
+This documentation provides details on how to access the Prisma Data Platform using the Prisma CLI.
+
+The integration, currently in [Early Access](/platform/maturity-levels#early-access), extends the functionality of the Prisma CLI, enabling you to manage your Platform projects with Prisma Accelerate directly from the CLI. For the complete list of available commands, navigate to the [Commands page](/platform/platfrom-cli/commands).
```

Comment:
I would drop this part: 

> enabling you to manage your Platform projects with Prisma Accelerate directly from the CLI

It may give the impression that the CLI is only relevant when using Accelerate.